### ISSUE TITLE
scribe, pagination, provenance

### ISSUE THREAD

**Issue by siglun** [schemaorg/schemaorg#2995]
I added a comment somewhere else, now I realize that what I should have added an issue. I copy the comment here.

* [Manuscripts](https://schema.org/Manuscript)  require [numberOfPages](https://schema.org/numberOfPages) (or perhaps even number of folios. Manuscripts are more often than not foliated not paginated),
* they are written by a `scribe` who may or may not be identical to the `author` (usually not identical for medieval manuscripts) 
* and  may have a  `provenance` (which is basically a list  of changes of `owner` [Events](https://schema.org/Event)). 

It is a problem that "manuscripts" are as wide a genre as "printed matter" or "web page".
https://github.com/schemaorg/schemaorg/issues/1448#issuecomment-979107885

**Comment by RichardWallis** [comment]
Thanks for your thoughts:

- One approach would be to use the [materialExtent](https://schema.org/materialExtent) property especially for folios etc.  There also could be a case for extending the domain of the _numberOfPages_ property to include _Manuscript_.
- Currently a scribe would be considered as a [contributor](https://schema.org/contributor).
- [OwnershipInfo](https://schema.org/OwnershipInfo) maybe a good starting point in thinking about the provenance issue you raise.  However, currently it is Person/Organization focussed. Applying it from the point of view of the thing being owned would need some consideration.  Who and how many might apply it not being the least consideration.

**Comment by giacomomarchioro** [comment]
Hi, @siglun @RichardWallis is there any progress on this spec? I was looking for a reference implementation. We were thinking of converting the number of folios into pages. Regarding the provenance, we were thinking of using https://schema.org/locationCreated and https://schema.org/countryOfOrigin

**Comment by siglun** [comment]
Nope. At least I haven't done anything. Here is a proof of concept encoding of metadata for a Hebre manuscript:

https://github.com/kb-dk/ds-solr/blob/nested-solr-indexing/ld-json/responsa.json

which is transformed from the data for this one

http://www5.kb.dk/manus/judsam/2009/sep/dsh/object41158/en/

I have put in a relator name space 

```
"@context": [
        "http://schema.org/",
        {
          "kb": "http://kb.dk/vocabs/",
          "relator": "https://id.loc.gov/vocabulary/relators/"
        }
      ],

```
such that a `scribe` will here


```
"relator:scr": [
        {
          "@type": "Person",
          "name": {
            "@language": "en",
            "@value": "Yehuda Mordekhai ben Yitsḥaq Mordekhai"
          }
        }
```
Don't know if this is **good** solution. But it is a solution. Nothing of this is in production yet.

Thanks for asking!

Yours

Sigfrid

**Comment by siglun** [comment]
We do not have anything in production where we could (at least experimentally) add schema.org to the landing pages. And because of the priorities here I doubt it will be anything during 2023. But who knows.
